python/ray/dashboard/modules/state/state_head.py

```diff
@@ -273,6 +278,20 @@ async def get_logs(self, req: aiohttp.web.Request):
         output_format = req.query.get("format", "leading_1")
         logger.info(f"Streaming logs with format {output_format} options: {options}")
 
+        async def get_actor_fn(actor_id: str) -> dict:
+            # can't use self._state_api.list_actors because it filters out workerId and
+            # rayletId.
+            reply = await self._state_api_data_source_client.get_all_actor_info(
+                filters=[("actor_id", "=", actor_id)]
+            )
+            if len(reply.actor_table_data) == 0:
+                return None
+            assert len(reply.actor_table_data) == 1
+            return protobuf_message_to_dict(
```

Comment:
why do we need to covert to dict? the caller can just use the protobuf right which has the schema so better than dict.